apex/contrib/csrc/xentropy/interface.cpp

```diff
@@ -46,7 +48,16 @@ at::Tensor softmax_xentropy_backward(
     return softmax_xentropy_backward_cuda(grad_loss, logits, max_log_sum_exp, labels, smoothing);
 }
 
+std::string get_version() {
+#ifdef XENTROPY_VER
+    // return std::to_string(XENTROPY_VER);
+    return XENTROPY_VER;
+#endif
+    return {};
+}
+
 PYBIND11_MODULE(TORCH_EXTENSION_NAME, m) {
     m.def("forward", &softmax_xentropy_forward, "Softmax cross entropy loss with label smoothing forward (CUDA)");
     m.def("backward", &softmax_xentropy_backward, "Softmax cross entropy loss with label smoothing backward (CUDA)");
+    m.def("module_version", &get_version);
```

Comment:
It's a common practice to use `module.__version__` for storing the version information. This PEP 396 has detailed guidance on how it should look in general: https://peps.python.org/pep-0396/#specification.

Let's set the `__version__` attribute instead of defining a new function. I think the following suggestion should work:
```suggestion
    m.attr("__version__", get_version());
```

Reply:
thanks, now I see:
```
❯ python -c 'import torch; import xentropy_cuda; print(f"{xentropy_cuda.__version__=}")'
xentropy_cuda.__version__='23.09.03'
```